test/quantization/fx/test_quantize_fx.py

```diff
@@ -596,6 +596,70 @@ def conv_bn_res_relu_extra_inputs_getter(pattern):
             if node.op == "call_module" and type(named_modules[node.target]) == torch.nn.Conv2d:
                 self.assertTrue(len(node.args) == 2), "Expecting the fused op to have two arguments"
 
+    def test_fusion_pattern_with_matchallnode(self):
+        """This test tests that the node matched by MatchAllNode will be regared as an input
```

Comment:
thanks for the comment, can we add a bit more description about how this is tested?